htmresearch/frameworks/layers/l2_l4_inference.py

```diff
@@ -578,6 +580,42 @@ def getL2Representations(self):
     return [set(column._pooler.getActiveCells()) for column in self.L2Columns]
 
 
+  def getCurrentClassification(self, minOverlap=None):
+    """
+    A dict with a score for each object. Score goes from 0 to 1. A 1 means
+    every col (that has received input since the last reset) currently has
+    overlap >= minOverlap with the representation for that object.
+
+    :param minOverlap: min overlap to consider the object as recognized.
+                       Defaults to half of the SDR size
+    :return: dict of object names and their score
+    """
+    results = {}
+    l2sdr = self.getL2Representations()
+    sdrSize = self.config["L2Params"]["sdrSize"]
+    if minOverlap is None:
+      minOverlap = sdrSize / 2
+
+    for objectName, objectSdr in self.objectL2Representations.iteritems():
+      count = 0
+      score = 0.0
+      for i in xrange(self.numColumns):
+        # Ignore inactive column
+        if len(l2sdr[i]) == 0:
+          continue
+
+        count += 1
+        overlap = len(l2sdr[i] & objectSdr[i])
+        if overlap >= minOverlap:
+          score += overlap
```

Comment:
I think this should just be += 1. The number of bits doesn't matter once it's above a threshold.